fix(curriculum): add description for C# exam (#50779)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam.md

# --description--

> Pass this exam to earn your Foundational C# with Microsoft Certification. Please familiarize yourself with the following guidelines and click the start button when you are ready to begin:

Reviewer: ```suggestion
To earn your Foundational C# with Microsoft Certification, you need to pass this exam. Before starting the exam, please review the following guidelines:
```

> - You must complete the exam in a single session.

Reviewer: ```suggestion
- You must complete the entire exam in one session.
```

> - If you exit the exam at any point, it will result in the loss of your progress.

Reviewer: ```suggestion
- Exiting the exam at any point will result in the loss of your progress.
```

> - The exam will comprise of multiple-choice questions.

Reviewer: ```suggestion
- The exam consists of multiple-choice questions.
```

> - There is no time limit, but your time will be recorded.

Reviewer: ```suggestion
- There is no specific time limit, but your total time taken will be recorded.
```

> - Complete the exam and answer a sufficent number of question correctly to qualify to claim your certification.

Reviewer: ```suggestion
- To qualify for certification, you need to complete the exam and answer a sufficient number of questions correctly.
```
